### PR TITLE
itests: payments test flake fix

### PR DESCRIPTION
**Pull request by bhandras**
This issue has been fixed before in https://github.com/lightningnetwork/lnd/pull/5579 but it seems like one more left flaking occasionally.

### CODE REVIEW

File: lntest/itest/lnd_payment_test.go [hunk 1]
```diff
@@ -581,24 +581,28 @@ func testBidirectionalAsyncPayments(net *lntest.NetworkHarness, t *harnessTest)
...
-	if bobInfo.RemoteBalance != aliceAmt {
-		t.Fatalf("bob's remote balance is incorrect, got %v, "+
-			"expected %v", bobInfo.RemoteBalance, aliceAmt)
-	}
-	if len(bobInfo.PendingHtlcs) != 0 {
-		t.Fatalf("bob's pending htlcs is incorrect, got %v, "+
-			"expected %v", len(bobInfo.PendingHtlcs), 0)
-	}
+		if bobInfo.LocalBalance != bobAmt {
+			t.Fatalf("bob's local balance is incorrect, got %v, "+
```
Comment (carlaKC) [inline lntest/itest/lnd_payment_test.go:592]: if we `t.Fatalf` in every check, won't think only run once anyway? Would thing we need to return these errors rather

Comment (bhandras) [reply lntest/itest/lnd_payment_test.go]: Thanks, somehow forgot to update these to `Errorf`... Now fixed :sweat_smile:

**Review by yyforyongyu — APPROVED** [review]
LGTM👍

**Review by carlaKC — APPROVED** [review]
Very nice!

**Pull request merged**